web-components/src/[sandbox]/examples/tabs.ts

```diff
@@ -293,7 +294,7 @@ export const tabsTemplate = html`
           <span>Content for "Contact History"</span>
         </md-tab-panel>
         <md-tab slot="tab" name="WxM" closable="auto" label="WxM">
-          <md-icon name="apps_16"></md-icon>
+          <span style="height: 16px; width: 16px"><img src="${svgWxm}"/></span>
```

Comment:
Stick with `rem` values everywhere possible.
Also this repeats, so a class style is preferable to apply to all spans that need this style

Reply:
This is just inside the sandbox demo. Not in our component. In the component I have taken care properly. @futurethang